nanoc-live/spec/nanoc/live/live_recompiler_spec.rb

```diff
@@ -124,7 +124,7 @@
 
     sleep 1.0 # HFS+ mtime resolution is 1s
     File.write('lib/lol.rb', 'def greeting; "yo"; end')
-    sleep 0.1 until File.read('output/lol.html') == 'yo'
+    sleep 0.1 until File.file?('output/lol.html') && File.read('output/lol.html') == 'yo'
```

Comment:
`clonefile` requires the target not to exist, so the target can disappear before it is updated.